Sources/KeyboardSupport/KeyboardToolbar.swift

```diff
@@ -78,6 +92,52 @@ open class KeyboardToolbar: UIToolbar, KeyboardAccessory {
     }
 }
 
+// MARK: - Private helpers
+private extension KeyboardToolbar {
+    private func storeButton(_ button: UIBarButtonItem, ofType type: ButtonNavigationType) {
+        switch type {
+        case .back:
+            backButton = button
+        case .next:
+            nextButton = button
+        case .done:
+            doneButton = button
+        }
+    }
+}
+
+// MARK: - methods for hiding next and back buttons
+public extension KeyboardToolbar {
+    public func setNextAndBackButtonsHidden(hidden: Bool) {
```

Comment:
This could read a little better at the call site if we used an unnamed parameter.
```swift
public func setNextAndBackButtonsHidden(_ hidden: Bool) {
```